### ISSUE TITLE
Feature Request: Re-focus to text area after inserting

### ISSUE THREAD

**Issue by Tristan-Goucher** [schultzcole/FVTT-Autocomplete-Inline-Properties#14]
It's a minor thing, but it builds up the longer I'm working at content. When I get down to the property I want and hit Enter, I have to back and manually click on the text area I was in to finish working on my formula every time. It'd be a much smoother workflow if I could hit Enter and then keep on going, so-to-speak.

**Comment by schultzcole** [comment]
Yep, this is definitely something I want to add when I get the chance, if I can.
